### ISSUE TITLE
Return boolean from modal

### ISSUE THREAD

**Issue by JoshPJackson** [mkocansey/bladewind#145]
I'm trying to show a modal when a user tries to submit a form. If they press ok, then the form should submit. If they press cancel, then the modal should just close. Is there a way to have the ok action return true and the cancel action return false?

What I'm trying:
```
<form>

    <button type="submit" onclick="return showModal('dosomething')">Save</button>

    <x-bladewind.modal
            name="dosomething"
            title="Does something"
            type="info"
            ok-button-action="() => true"
            cancel-button-action="() => false"
    >
        Are you sure?
    </x-bladewind.modal>

</form>
```

**Comment by mkocansey** [comment]
Hey @JoshPJackson yes it is possible to achieve this using the code below.

```
<form class="test-form">

    <button type="button" onclick="showModal('dosomething')">Save</button>

    <x-bladewind.modal
            name="dosomething"
            title="Does something"
            type="info"
            ok_button_action ="domEl('.test-form').submit()">
        Are you sure?
    </x-bladewind.modal>

</form>
```

From the code above
1. I added `class='test-form'` to your form so we will be able to target it from Javascript.
2. I changed your Save button from `type="submit"` to `type="button"` to prevent the button from submitting the form when it is clicked. I also removed the `return` keyword.
3. I added `ok_button_action ="domEl('.test-form').submit()"` to the modal to tell the Ok button to submit the form. `domEl()` is a helper function available in BladewindUI that is basically a shortcut for `document.querySelector`. Note: the attribute uses underscore (`ok_button_action `) and not dashes like you had (`ok-button-action `).
4. I removed the action for the Cancel button because by default it closes the modal.

I have not tested this very specific code but I use this similar approach in my projects and it works. Let me know if this works for you.
